Sources/Time/2. Calendar Core/Region.swift

```diff
@@ -44,4 +44,9 @@ public struct Region: Hashable {
         self.locale = locale
     }
     
+    /// The `Instant` of the next daylight saving time transition, or `nil` if the time zone does not currently observe daylight saving time.
+    public var nextDaylightSavingTimeTransition: Instant? {
+        timeZone.nextDaylightSavingTimeTransition.map(Instant.init)
+    }
+    
```

Comment:
Seeing this here... I'm thinking that this would actually make more sense as a method (#7) on `Clock`. This method assumes that you know when "now" is (otherwise how would you know what "next" means), and technically a `Clock` is the only thing that knows what `now` is. How about something like:

```swift
extension Clock {
    public func nextDaylightSavingTimeTransition(after instant: Instant? = nil) -> Instant?
}
```

Specifying `after: nil` would assume the current time, according to the clock's `now()` method.

Reply:
Existing on `Clock` makes a lot more sense than `Region` – thanks for the feedback!